Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocRowRenderer.cs

```diff
@@ -181,6 +197,39 @@ private IBocIndexColumnRenderer GetIndexColumnRenderer ()
       return _indexColumnRenderer;
     }
 
+    private void RenderValidationRow (
+        BocListRenderingContext renderingContext,
+        BocListRowRenderingContext rowRenderingContext,
+        BocListDataRowRenderEventArgs rowRenderEventArgs,
+        IReadOnlyCollection<BocListValidationFailureWithLocationInformation> validationFailuresWithLocationInfo)
+    {
+      var cssClassValidationRow = GetCssClassValidationRow(rowRenderingContext.IsSelected, rowRenderEventArgs);
+
+      renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Class, cssClassValidationRow);
+      renderingContext.Writer.RenderBeginTag(HtmlTextWriterTag.Tr);
+
+      // The BocList also contains two columns not existent in the ColumnRenderers property (the number and selection column)
+      // Therefore we need to add these to the count.
+      var totalColumnsCount = renderingContext.ColumnRenderers.Length + 2;
+
+      //QUESTION: Is there a better way to do this? saying int.toString() feels so wrong.
+      renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Colspan, totalColumnsCount.ToString());
+      renderingContext.Writer.RenderBeginTag(HtmlTextWriterTag.Td);
+
+      renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Class, "bocListValidationRowStickyContainer");
```

Comment:
Classes are usually only applied to "top-level-like" elements and not used as marker for containers inside the control. This is something that needs to be discussed with @MichaelKetting

Reply:
oh, okay. yeah, in that case it's not really needed or helpful.